src/ai/backend/manager/models/image.py

```diff
@@ -272,6 +282,29 @@ async def from_alias(
         else:
             raise UnknownImageReference
 
+    @classmethod
+    async def from_image_identifier(
+        cls,
+        session: AsyncSession,
+        identifier: ImageIdentifier,
+        load_aliases: bool = True,
+    ) -> ImageRow:
+        query = sa.select(ImageRow).where(
+            ImageRow.name == identifier.canonical
+            and ImageRow.architecture == identifier.architecture
+        )
```

Comment:
```suggestion
        query = sa.select(ImageRow).where(
            (ImageRow.name == identifier.canonical)
            and (ImageRow.architecture == identifier.architecture)
        )
```
As I remember, trailing condition statement (like `Row.id == row_id and Row.name == row_name`) raises syntax error.
This would not raise error since two condition statements are divided by a new line but let's make it safe just in case.